styx-scheduler-service/src/test/java/com/spotify/styx/api/SchedulerResourceTest.java

```diff
@@ -335,13 +336,17 @@ public void testTriggeredWorkflowGeneratesTrigger() throws Exception {
     TriggerRequest toTrigger = TriggerRequest.of(HOURLY_WORKFLOW.id(), "2014-12-31T23");
 
     Response<ByteString> response = requestAndWaitTriggerWorkflowInstance(toTrigger);
-
+    final TriggerResponse triggerResponse =
+        OBJECT_MAPPER.readValue(response.payload().get().toByteArray(), TriggerResponse.class);
 
     assertThat(response.status(), is(Status.OK));
     final Instant expectedInstant = Instant.parse("2014-12-31T23:00:00.000Z");
     verify(triggerListener).event(eq(HOURLY_WORKFLOW), triggerCaptor.capture(), eq(expectedInstant), eq(expectedParameters));
-    assertThat(TriggerUtil.triggerType(triggerCaptor.getValue()), is("adhoc"));
-    assertThat(TriggerUtil.triggerId(triggerCaptor.getValue()), startsWith("ad-hoc-cli-"));
+    final Trigger trigger = triggerCaptor.getValue();
+    final String triggerId = TriggerUtil.triggerId(trigger);
+    assertThat(TriggerUtil.triggerType(trigger), is("adhoc"));
```

Comment:
Maybe taking the opportunity to change `assertThat` to `import static org.hamcrest.MatcherAssert.assertThat;`? `org.junit.Assert.assertThat` is deprecated.

Reply:
There are ~60 other uses of this import. I fixed this one and can fix the rest in another PR.

Reply:
Yeah I was only referring the file we anyway need to touch in this PR. The rest can wait.